### PR TITLE
Close #4550: All tables and figures without ``align`` option are displayed to center

### LINKED ISSUE #4550: Tables should be centered by default.
In general having all tables pushed to the left is an undesirable behaviour. 
Tables should be centred by default and the user should be able to align them the the left or right if desired.

See #3942

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Feature

### Purpose
- refs: #4550

### CODE REVIEW

File: sphinx/io.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from sphinx.deprecation import RemovedInSphinx30Warning
 from sphinx.transforms import (
     ApplySourceWorkaround, ExtraTranslatableNodes, CitationReferences,
-    DefaultSubstitutions, MoveModuleTargets, HandleCodeBlocks, SortIds,
+    DefaultSubstitutions, MoveModuleTargets, HandleCodeBlocks, SortIds, FigureAliner,
```
Comment (jfbu) [inline sphinx/io.py]: Perhaps `FigureAligner` is better spelling? :)

File: sphinx/transforms/__init__.py [hunk 2]
```diff
@@ -309,6 +309,19 @@ def apply(self, **kwargs):
                                location=node)
 
 
+class FigureAliner(SphinxTransform):
```
Comment (jfbu) [inline sphinx/transforms/__init__.py]: see above, ``FigureAligner()``

Comment (tk0miya) [reply sphinx/transforms/__init__.py]: Oops...

**Comment by tk0miya** [comment]
Merged.

**Pull request merged**